cmd/server/assets/apikeys/show.html

```diff
@@ -86,8 +86,8 @@ <h1>{{$authApp.Name}} API key</h1>
         </span>
         <span>
           <span class="mr-1">Export as:</span>
-          <a href="/stats/realm/api-keys/{{$authApp.ID}}.csv" class="mr-1">CSV</a>
-          <a href="/stats/realm/api-keys/{{$authApp.ID}}.json">JSON</a>
+          <a href="/stats/realm/api-keys/{{$authApp.ID}}.csv" target="_blank" class="mr-1">CSV</a>
+          <a href="/stats/realm/api-keys/{{$authApp.ID}}.json" target="_blank">JSON</a>
```

Comment:
Why target blank? The CSV should trigger as a file download.

Reply:
That's true it may be redundant for CSV, but for JSON it takes over the window